sheet_to_triples/__main__.py

```diff
@@ -88,6 +88,13 @@ def parse_args(argv):
         help=('Which triples to keep when dropping duplicates during '
               'model normalisation'),
         )
+    parser.add_argument(
+        '-n', '--use-ontology-normalisation', action='store_true',
+        help=('Unique predicates are defined by owl:functionalProperty in the '
+              'ontology and used for model normalisation. All non-unique '
+              'definitions are ignored, as all predicates are assumed non-unique '
+              ' by default unless specifically defined as unique.')
```

Comment:
How does this interact with --non-unique-from?

Reply:
`--non-unique-from` may still be passed and it will still retrieve the non-unique predicates from a target set of transforms, but when we get to `normalise_model` there's a switch. If `--use-ontology-normalisation` is not set, then we use `non_uniques` and the existing behaviour. If it *is* set, though, then we don't use `non_uniques` for normalisation, we instead use a new list of predicates provied by the `_functional_properties` function. `non_uniques` is never used.

So there's no restrictions around passing `--non-unique-from` and `--use-ontology-normalisation` at the same time, but `--use-ontology-normalisation` will always win - if it is passed, `non_uniques` and all methods of building it are ignored in favour of the predicates definitions in the ontology.